coriolis/providers/replicator.py

```diff
@@ -470,10 +470,13 @@ def _get_ssh_client(self, args):
         """
         gets a paramiko SSH client
         """
-        ssh = paramiko.SSHClient()
-        ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        ssh.connect(**args)
-        return ssh
+        try:
+            ssh = paramiko.SSHClient()
+            ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+            ssh.connect(**args)
+            return ssh
+        except paramiko.ssh_exception.SSHException as ex:
+            raise NotAuthorized("Failed to setup SSH client: %s" % str(ex))
```

Comment:
Why would any ssh exception be related to authorization issues?

Reply:
I asked him to pick this one (I had the "Authentication failed" message in mind), but you're right, we should use the more generic `CoriolisException` one. The gist of this PR is to present a more friendly message when SSH exceptions occur. For example, "Authentication failed." just printed out in the task failure message is pretty vague, we need to specify this is a SSH client error. 

Therefore, @OneBuildMan please fix the import statement above, and change the exception type to a `CoriolisException`.